suite/tests/client-interface/drwrap-drreg-test.dll.c

```diff
@@ -80,7 +80,12 @@ module_load_event(void *drcontext, const module_data_t *mod, bool loaded)
 
         addr_two_args = (app_pc)dr_get_proc_address(mod->handle, "two_args");
         CHECK(addr_two_args != NULL, "cannot find lib export");
+#if defined(RISCV64)
+        bool ok = drwrap_wrap_ex(addr_two_args, wrap_pre, wrap_post, 0,
+                                 DRWRAP_CALLCONV_RISCV_LP64 | DRWRAP_REPLACE_RETADDR);
```

Comment:
Please change the DRWRAP_CALLCONV_DEFAULT to be DRWRAP_CALLCONV_RISCV_LP64: we should not have to pass that explicitly every single time on RISC-V.

Reply:
Set DRWRAP_CALLCONV_DEFAULT for RISCV